src/main/java/org/gridsuite/dynamicsecurityanalysis/server/controller/DynamicSecurityAnalysisController.java

```diff
@@ -138,4 +141,13 @@ public ResponseEntity<List<String>> getProviders() {
     public ResponseEntity<String> getDefaultProvider() {
         return ResponseEntity.ok().body(dynamicSecurityAnalysisService.getDefaultProvider());
     }
+
+    @GetMapping(value = "/results/{resultUuid}/download/debug-file", produces = "application/json")
```

Comment:
Like https://github.com/gridsuite/dynamic-simulation-server/pull/117#discussion_r2154485878 I guess : 
```suggestion
    @GetMapping(value = "/results/{resultUuid}/download-debug-file", produces = "application/json")
```

Reply:
DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/19/commits/6cf7fd5c6ffcec7a6cd751dbec3762bdf28e49f8